modules/tfm/Kconfig

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+config TFM_BOARD
+	string
+	default "${ZEPHYR_NRF_MODULE_DIR}/modules/tfm/boards/nrf9160" if SOC_NRF9160_SICA
+	default "${ZEPHYR_NRF_MODULE_DIR}/modules/tfm/boards/nrf5340_cpuapp_eng_a" if SOC_NRF5340_CPUAPP_QKAA && NRF5340_CPUAPP_ERRATUM19
+	default "${ZEPHYR_NRF_MODULE_DIR}/modules/tfm/boards/nrf5340_cpuapp" if SOC_NRF5340_CPUAPP_QKAA && !NRF5340_CPUAPP_ERRATUM19
```

Comment:
@ioannisg whatever happened to this:
> I did not read the whole conversation but I lean towards accepting the latest proposal. I'll open a PR and we can finalize the discussion there

https://github.com/zephyrproject-rtos/zephyr/pull/30062#discussion_r526716896

I think now is a good time to actually cleanup this, and remove the use of paths from Kconfig.tfm.

Depending on the order in which we take in PRs, I can also add this cleanup as part of this PR: https://github.com/zephyrproject-rtos/zephyr/pull/30905

Is this PR expected to enter 1.5 ? (it's not milestoned)

Reply:
guess this will happen as part of this: https://github.com/zephyrproject-rtos/zephyr/pull/30145
and thus probably waits for this: https://github.com/zephyrproject-rtos/zephyr/pull/30905